client/src/store/list-data/intl-jurisdictions.ts

```diff
@@ -1,1248 +1,1248 @@
-    "text": "SAINT HELENA"
+    "SHORT_DESC": "Saint Helena",
+    "text": "Saint Helena"
   },
   {
     "value": "KN",
-    "SHORT_DESC": "SAINT KITTS AND",
-    "text": "SAINT KITTS AND NEVIS"
+    "SHORT_DESC": "Saint Kitts And",
+    "text": "Saint Kitts And Nevis"
   },
   {
     "value": "LC",
-    "SHORT_DESC": "SAINT LUCIA",
-    "text": "SAINT LUCIA"
+    "SHORT_DESC": "Saint Lucia",
+    "text": "Saint Lucia"
   },
   {
     "value": "PM",
-    "SHORT_DESC": "SAINT PIERRE AN",
-    "text": "SAINT PIERRE AND MIQUELON"
+    "SHORT_DESC": "Saint Pierre An",
+    "text": "Saint Pierre And Miquelon"
   },
   {
     "value": "VC",
-    "SHORT_DESC": "SAINT VINCENT A",
-    "text": "SAINT VINCENT AND THE GRENADINES"
+    "SHORT_DESC": "Saint Vincent A",
+    "text": "Saint Vincent And The Grenadines"
   },
   {
     "value": "WS",
-    "SHORT_DESC": "SAMOA",
-    "text": "SAMOA"
+    "SHORT_DESC": "Samoa",
+    "text": "Samoa"
   },
   {
     "value": "SM",
-    "SHORT_DESC": "SAN MARINO",
-    "text": "SAN MARINO"
+    "SHORT_DESC": "San Marino",
+    "text": "San Marino"
   },
   {
     "value": "ST",
-    "SHORT_DESC": "SAO TOME AND PR",
-    "text": "SAO TOME AND PRINCIPE"
+    "SHORT_DESC": "Sao Tome And Pr",
+    "text": "Sao Tome And Principe"
   },
   {
     "value": "SA",
-    "SHORT_DESC": "SAUDI ARABIA",
-    "text": "SAUDI ARABIA"
+    "SHORT_DESC": "Saudi Arabia",
+    "text": "Saudi Arabia"
   },
   {
     "value": "SN",
-    "SHORT_DESC": "SENEGAL",
-    "text": "SENEGAL"
+    "SHORT_DESC": "Senegal",
+    "text": "Senegal"
   },
   {
     "value": "SC",
-    "SHORT_DESC": "SEYCHELLES",
-    "text": "SEYCHELLES"
+    "SHORT_DESC": "Seychelles",
+    "text": "Seychelles"
   },
   {
     "value": "SL",
-    "SHORT_DESC": "SIERRA LEONE",
-    "text": "SIERRA LEONE"
+    "SHORT_DESC": "Sierra Leone",
+    "text": "Sierra Leone"
   },
   {
     "value": "SG",
-    "SHORT_DESC": "SINGAPORE",
-    "text": "SINGAPORE"
+    "SHORT_DESC": "Singapore",
+    "text": "Singapore"
   },
   {
     "value": "SK",
-    "SHORT_DESC": "SLOVAKIA",
-    "text": "SLOVAKIA"
+    "SHORT_DESC": "Slovakia",
+    "text": "Slovakia"
   },
   {
     "value": "SI",
-    "SHORT_DESC": "SLOVENIA",
-    "text": "SLOVENIA"
+    "SHORT_DESC": "Slovenia",
+    "text": "Slovenia"
   },
   {
     "value": "SB",
-    "SHORT_DESC": "SOLOMON ISLANDS",
-    "text": "SOLOMON ISLANDS"
+    "SHORT_DESC": "Solomon Islands",
+    "text": "Solomon Islands"
   },
   {
     "value": "SO",
-    "SHORT_DESC": "SOMALIA",
-    "text": "SOMALIA"
+    "SHORT_DESC": "Somalia",
+    "text": "Somalia"
   },
   {
     "value": "ZA",
-    "SHORT_DESC": "SOUTH AFRICA",
-    "text": "SOUTH AFRICA"
+    "SHORT_DESC": "South Africa",
+    "text": "South Africa"
   },
   {
     "value": "GS",
-    "SHORT_DESC": "SOUTH GEORGIA A",
-    "text": "SOUTH GEORGIA AND THE SOUTH SANDWICH ISL"
+    "SHORT_DESC": "South Georgia A",
+    "text": "South Georgia And The South Sandwich Isl"
```

Comment:
A -> a
And The -> and then

(I'm Googling these proper nouns to get the correct capitalization.)